.github/workflows/take.yml

```diff
@@ -0,0 +1,18 @@
+# .github/workflows/take.yml 
+name: Assign issue to contributor
+on: 
+  issue_comment:
+
+jobs:
+  assign:
+    name: Take an issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - name: take the issue
+      uses: bdougie/take-action@main
+      with:
+        message: Thanks for taking this issue! Let us know if you have any questions!
+        trigger: .take
```

Comment:
You know, on second glance, this should probably be documented in the repo's development docs. @blackgirlbytes do you want to do that in this PR or a followup?

Reply:
in a follow up if that's okay!